src/components/application_manager/include/application_manager/state_controller_impl.h

```diff
@@ -361,15 +361,6 @@ class StateControllerImpl : public event_engine::EventObserver,
    */
   void SetupRegularHmiState(ApplicationSharedPtr app, HmiStatePtr state);
 
```

Comment:
@EKuliiev For the future, please try to give mo narrow and descriptive commit names, with adding what was wrong and ehat actually did you fix.